test/unit/ngsiv2/contextRequests/updateStatus2.json

```diff
@@ -0,0 +1,15 @@
+{
+	"actionType": "APPEND",
```

Comment:
Action types are lowercase in NGSIv2 (uppercase are also allowed, but are deprecated). Thus, I'd suggest to use "append" here.

Reply:
...which probably needs a modification in the agent or library code.

In fact, I wonder why append is used here. I mean, if the entity already exist (and it should be, otherwise a command cannot be send on it) it should be "update" instead of "append". However, if we are unsure of how this works, let's keep "append" (which is always "safer").

Reply:
Fixed in library with 2f7eff9e1bdf07be79dbae46eca9f2e0e0851a1a

Reply:
Fixed here with 669a9c3053f0af12d6947225cf9a9f863f8fdec5